### PR TITLE
#35 update readme

### PR DESCRIPTION
**Pull request by koalalorenzo**
- Moves into different directories the docs and static files (`./docs`)
- Ensures links are pointing to github and not gitlab
- renames building cmds (TBV)
- updates the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,50 @@
...
 
-## How Help
+## Learn more
+You can find more documentation in the following pages:
 
+ * [How to run Lumpy](https://github.com/Siderus/Lumpy/tree/master/docs/RUN.md)
+ * [How to build Lumpy standalone App](https://github.com/Siderus/Lumpy/tree/master/docs/BUILD.md)
+ * ...More in [docs directory](https://github.com/Siderus/Lumpy/tree/master/docs)
+
+## How Help
```
Comment (0x6431346e) [inline README.md]: How to help?

File: README.md [hunk 2]
```diff
@@ -1,66 +1,50 @@
...
+removing files from the node using an GUI instead of CLI or web interface.
 
 The main goals of this project are:
 
-* explains and **educates** the user about the perks of using IPFS
-* shows only what is **important** to the user (Sometimes the name of the
-file/Directory, not hash. KISS, but configurable)
-* Help users to achieve their goals using IPFS instead of traditional tools.
+* Help embracing IPFS by providing an easy to use and intuitive interface.
+* Explains and **educates** the user about the perks of using IPFS
```
Comment (0x6431346e) [inline README.md]: Explain and educate?

File: README.md [hunk 3]
```diff
@@ -1,66 +1,50 @@
...
+* shows only what is **important** to the user (Ex: the name of the
```
Comment (0x6431346e) [inline README.md]: Show

**Comment by koalalorenzo** [comment]
Updated, Plz review! 🙏

**Pull request merged**